_posts/_posts/2024-03-18-4th-issue-newsletter.md

```diff
@@ -0,0 +1,248 @@
+
+Best regards,  [The OBO Foundry Operations Committee.](https://obofoundry.org/docs/OperationsCommittee.html)
+
+
+## Highlights 
+
+### OBO Dashboard Compliance is Live Now
+
+With the arrival of 2024, mandatory compliance with the OBO Dashboard is now in effect. The OBO Foundry will soon be showing a visual indicator on the homepage to display each ontology's compliance status. Non-compliant ontologies will be sorted and shaded at the bottom of their default ontology-by-topic view group. This will serve as an informative measure for users and encourages ontology curators to strive for quality control.
+
+Remember, only dashboard errors (black X on a red background) cause non-compliance. Warnings (yellow triangle) or information (blue i) in the final 'Summary' column are still acceptable. Ontologies in the dashboard are sorted by Dashboard status, which means that ontologies without errors are shown first: [https://dashboard.obofoundry.org/dashboard/index.html](https://dashboard.obofoundry.org/dashboard/index.html).
+
+An explanation of the dashboard report is available here ([link to dashboard video](https://www.youtube.com/watch?feature=shared&t=1319&v=m2khZcJVKU0)); more information can be found at [http://dashboard.obofoundry.org/dashboard/about.html](https://dashboard.obofoundry.org/dashboard/about.html).
+
+---
+
+## Decisions Made and Important Updates
+
+* Addition of a new ontology status: “[unresponsive](https://obofoundry.org/docs/OntologyStatus.html)”, which signifies that the ontology project does not have a contact person who is responsive, but the ontology is still being actively maintained. 
+* New Ontology submissions no longer need to pass the criterion “resolvable version IRI” during the submission process, as this is tied to the PURL resolution system. It is, however, expected that when an ontology is officially admitted, the version IRI does resolve correctly,  
+* Updating the [SOP document](https://obofoundry.org/docs/SOP.html) to include a new subsection under "Reviewing Ontologies for OBO Membership." This subsection, titled "Rules of Communication," designates the official ontology reviewer as the authority to decide which community suggestions need to be addressed and which do not, aiming to manage situations where community input might not meet the established principles. This addition is intended to streamline the review process and clarify the role of community feedback in ontology evaluations. For more details, please visit the pull request page #[2528](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2528)
+* EWG improved the OBO Foundry ontology review process by specifying how community input should be taken into account during the review process. The Ontology reviewer has the last word, and is responsible to communicate which feedback is optional, and which mandatory to be addressed before admission (Issue #[2459](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2459)).
```

Comment:
In the next item you say "The EWG"; I would make this one the same (add "The").

Reply:
```suggestion
* The EWG improved the OBO Foundry ontology review process by specifying how community input should be taken into account during the review process. The Ontology reviewer has the last word, and is responsible to communicate which feedback is optional, and which mandatory to be addressed before admission (Issue #[2459](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2459)).
```